radiomessages/exploration.radiomessages.patch

```diff
@@ -10,7 +10,7 @@
     "op": "replace"
   },
   {
-    "value": "A madeira \u00E9 essencial para sua sobreviv\u00EAncia. Voc\u00EA ser\u00E1 capaz de fabricar ^orange;Armas B\u00E1sicas^reset; e equipamentos com suas pr\u00F3prias m\u00E3os. Eu recomendo criar ^green;Primitive Spear^reset; no come\u00E7o do jogo. ^orange;Pesquise^reset; outros recursos para a madeira no seu ^orange;Personal Tricorder^reset;.",
+    "value": "A madeira \u00E9 essencial para sua sobreviv\u00EAencia. Voc\u00EA ser\u00E1 capaz de fabricar ^orange;Armas B\u00E1sicas^reset; e equipamentos com suas pr\u00F3prias m\u00E3os. Eu recomendo criar ^green;Primitive Spear^reset; no come\u00E7o do jogo. ^orange;Pesquise^reset; outros recursos para a madeira no seu ^orange;Personal Tricorder^reset;.",
```

Comment:
`sobreviv\u00EAencia` pra `sobreviv\u00EAncia`

Reply:
tem mudar mano mais como?kkkk }
acho que vou pegar do zero mesmo e ir passando tudo ai após terminar 
antes de dar request eu te chamo dai a gente vai revisando junto

Reply:
porém farei amnha 
to de folga amanhã